ert/storage/_storage.py

```diff
@@ -8,7 +8,7 @@
 import requests
 
 import ert
-from ert_shared.asyncio import get_event_loop
+from ert_shared.async_utils import get_event_loop
```

Comment:
What error did this cause? Could be natural to describe the error in the commit. Maybe also refer to some issue in the PR.

Reply:
Only failed in snake_oil in snake_oil_simulator.py script in the run_ert2_test_data_setups.yml. It happens when it imports from the res package: from res.test.synthesizer import OilSimulator.

I am unsure why this started happening in my PR.

Reply:
How does the rename of the module resolve the error?

Reply:
Probably because it conflicted with asyncio in some way. I don't know. It was weird.

Reply:
I think a commit message body to 257080d697c7692639749f65a7a8a437c8284c94 describing the situation would be suffice :)